Packs/RedCanary/Integrations/RedCanary/RedCanary.py

```diff
@@ -80,7 +82,7 @@ def http_request(requests_func, url_suffix, **kwargs):
     if res.status_code not in [200, 201, ]:
         LOG('result is: %s' % (res.json(),))
         error = res.json()
-        raise Exception('Your request failed with the following error: {}.\n'.format(error, ))
+        raise Exception(f'Your request failed with the following error: {error}.\n')
```

Comment:
keep the `.format()` syntax across the integration instead of using f-strings.
it may raise problems in environments without docker (when using local python)